data/phi0474/phi009/__cts__.xml

```diff
@@ -1,19 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xml:lang="lat" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:phi0474" urn="urn:cts:latinLit:phi0474.phi009" projid="latinLit:phi009">
     
-    <ti:title xml:lang="eng">On Pompey's Command</ti:title>
+    <ti:title xml:lang="lat">De Imperio Cn. Pompei Ad Quirites</ti:title>
 
     
     <ti:edition urn="urn:cts:latinLit:phi0474.phi009.perseus-lat2" workUrn="urn:cts:latinLit:phi0474.phi009" xml:lang="lat">
         <ti:label xml:lang="lat">De Imperio Cn. Pompei Ad Quirites</ti:label>
-        <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Vol. I. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1908.</ti:description>
+        <ti:description xml:lang="mul">Cicero. M. Tulli Ciceronis. Orationes, Volume 1. Clark, Albert Curtis, editor. Oxford: Clarendon Press, 1908.</ti:description>
+        
     </ti:edition>
   
     
     <ti:translation urn="urn:cts:latinLit:phi0474.phi009.perseus-eng2" workUrn="urn:cts:latinLit:phi0474.phi009" xml:lang="eng">
         <ti:label xml:lang="eng">On Pompey’s Command</ti:label>
```

Comment:
Two different English titles

Reply:
I think that is a mistake here, thanks for catching.